lecture_notes/lesson_3.md

```diff
@@ -1,9 +1,13 @@
 # Lesson 3
+** Lesson goal:** At the end of this lesson the learner will understand the basic concepts of conditionals and Boolean algebra
```

Comment:
Nechceme je přece strašit nějakými termy.  :D
Na "Boolean algebra" budou mít na VŠ času dost. 
Stačí podle mě conditionals

Reply:
Tento termín se objevil už na řádce 58, proto jsem ho použil i tady.

Reply:
Přepíšu to na termín logical operators.